openlr/openlr_match_quality/openlr_assessment_tool/mainwindow.cpp

```diff
@@ -226,49 +231,76 @@ class PointsControllerDelegate : public PointsControllerDelegateBase
 };
 }  // namespace
 
-
-MainWindow::MainWindow(Framework & framework)
+MainWindow::MainWindow(Framework & framework, std::string const & url, std::string const & login,
+                       std::string const & paswd)
   : m_framework(framework)
 {
+
```

Comment:
Seems that you don't need this blank line?

Reply:
Done.